yt_dlp/extractor/facebook.py

```diff
@@ -524,6 +528,12 @@ def extract_metadata(webpage):
                     webpage, 'view count', default=None)),
                 'concurrent_view_count': get_first(post, (
                     ('video', (..., ..., 'attachments', ..., 'media')), 'liveViewerCount', {int_or_none})),
+                **traverse_obj(post_stats, {
+                    'like_count': ('likers', 'count', {int}),
+                    'comment_count': ('total_comment_count', {int}),
+                    'repost_count': ('share_count_reduced', {parse_count}),
+                    'reactor_count': ('unified_reactors', 'count', {int}),
```

Comment:
Oh, I think I didn't express myself well initially.
yt-dlp has a list of allowed values that can be returned as part of the info dict. https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/common.py#L104
`reactor_count` is not one of them. As yt-dlp is not intended to be a universal metadata scraper, we won't add additional site-specific fields that cannot be cleanly mapped to the generic fields already present.

`like_count`, `comment_count`, and `repost_count` can be added as part of this PR. If you require `reactor_count` for whatever reason, you should look in to using a plugin to extract that.
```suggestion
```